packages/components/src/site-thumbnail/use-mshots-img.tsx

```diff
@@ -38,7 +38,7 @@ export const useMshotsImg = (
 		[ src, options, retryCount ]
 	);
 	const imgRef = useRef< HTMLImageElement >( null );
-	const [ isLoading, setIsLoading ] = useState( true );
+	const [ isLoading, setIsLoading ] = useState( false );
```

Comment:
I'm a little bit confused about why `isLoading` is initially set to `true`. I'm not super sure about how this hook works, but wouldn't it mean that the loading state isn't shown until we get a `404` returned by the server? But what if there's never a `404` because we get the image back first try? Wouldn't we still want to have the loading visual while the image is downloading?

Reply:
I see, maybe we don't want the loading state to appear for that first time round because otherwise we see the fallback first letter ...

Reply:
setting loading to true initially shows the fallback state at first all times, removing it here was to set it in `onload` so that it would be fired when `onload` is called in the background but this cause the fallback state from earlier to be rendered at all. I think we should set is loading to true initially and also in the `onload` function to capture the background event scenario